alerts.py

```diff
@@ -222,11 +224,17 @@ def main():
             badRevisions = []
             # Reset the loop for upcoming stages
             alert['loop'] = 0
+            results = None
             for i in range(1, len(revisionList)):
                 results = compare(alert['test'], alert['buildername'], revisionList[i], revisionList[i-1])
                 if results < -2.0:
                     badRevisions.append(revisionList[i])
 
+            if results is None:
+                LOG.info("We did not find perf data for the revisions for alert %s on buildername %s. "
+                         "Will loop once again." % (alert['test'], alert['buildername']))
```

Comment:
should we continue in this loop if we hit this case 4 or 5 times in a row? I think 3 times should be a limit, but that is hard to tell.  Will we hit any other timeouts to terminate this cycle?